app/src/main/java/app/revanced/integrations/shared/Utils.java

```diff
@@ -196,18 +196,29 @@ public static float getResourceDimension(@NonNull String resourceIdentifierName)
         return getContext().getResources().getDimension(getResourceIdentifier(resourceIdentifierName, "dimen"));
     }
 
+    public interface MatchFilter<T> {
+        boolean matches(T object);
+    }
+
     /**
+     * @param searchRecursively If children ViewGroups should also be
+     *                          recursively searched using depth first search.
      * @return The first child view that matches the filter.
      */
     @Nullable
-    public static <T extends View> T getChildView(@NonNull ViewGroup viewGroup, @NonNull MatchFilter filter) {
+    public static <T extends View> T getChildView(@NonNull ViewGroup viewGroup, boolean searchRecursively,
```

Comment:
Instead of this new parameter, a separate method can be added that only searches the children.